service/src/main/java/org/jboss/sbomer/model/Sbom.java

```diff
@@ -165,6 +166,12 @@ public class Sbom extends PanacheEntityBase {
     @ToString.Exclude
     private JsonNode sbom;
 
+    @JsonIgnore
+    @Lob
+    @Type(type = "org.hibernate.type.TextType")
+    @Column(name = "task_run_msg")
+    private String taskRunMsg;
```

Comment:
I guess nitpicking, but I would not make this name related to TaskRun at all. For example calling it `message` or `statusMessage` would be better IMHO.

Reply:
right, looks much better